src/server/routers/tips.ts

```diff
@@ -44,16 +45,17 @@ export const tipsRouter = router({
             '1234567890abcdefghijklmnopqrstuvwxyz',
             5,
           )
-
-          const {instructor_id: instructorId} = JSON.parse(
-            ctx.req?.cookies.get('eh_user')?.value ?? "{instructor_id: ''}",
+          const {instructor_id: instructorId} = await loadCurrentUser(
+            ctx?.userToken,
           )
+          if (!instructorId) throw new Error('Instructor ID not found')
```

Comment:
providing more useful errors where production is currently giving this error which is opaque:

```
Caught TRPC error: SyntaxError: Unexpected token i in JSON at position 1
    at JSON.parse (<anonymous>)
```